G/GAP_pkg/GAP_pkg_kbmag/build_tarballs.jl

```diff
@@ -39,9 +38,7 @@ rm $prefix/lib/gap/sysinfo.gap
 
 name = gap_pkg_name(name)
 
-platforms = supported_platforms()
-filter!(p -> nbits(p) == 64, platforms) # we only care about 64bit builds
-filter!(!Sys.iswindows, platforms)      # Windows is not supported
+platforms = gap_platforms()
```

Comment:
Like `nq` this package only produces an executable and does not need GAP for this at all...